config/initializers/redis.rb

```diff
@@ -5,3 +5,4 @@
 APP_PROFILE_CACHE_PREFIX = "APP_PROFILE"
 APP_GROUP_PROFILE_CACHE_PREFIX = "APP_GROUP_PROFILE"
 GATE_GROUP_CACHE_PREFIX = "GATE_GROUP"
+APP_GROUP_REDACT_LABELS = "GROUP_REDACT_LABELS"
```

Comment:
I think this should be "APP_GROUP_REDACT_LABELS"

Reply:
Updated